api/buckets/service.go

```diff
@@ -1671,19 +1726,19 @@ func (s *Service) Archive(ctx context.Context, req *pb.ArchiveRequest) (*pb.Arch
 
 		// Include the existing archive in history,
 		// since we're going to set a new _current_ archive.
-		ffsi.Archives.History = append(ffsi.Archives.History, ffsi.Archives.Current)
+		ba.Archives.History = append(ba.Archives.History, ba.Archives.Current)
 	}
-	ffsi.Archives.Current = mdb.Archive{
+	ba.Archives.Current = mdb.Archive{
 		Cid:       p.Cid().Bytes(),
 		CreatedAt: time.Now().Unix(),
 		JobID:     jid.String(),
 		JobStatus: int(ffs.Queued),
 	}
-	if err := s.Collections.FFSInstances.Replace(ctx, ffsi); err != nil {
+	if err := s.Collections.BucketArchives.Replace(ctx, ba); err != nil {
 		return nil, fmt.Errorf("updating ffs instance data: %s", err)
 	}
 
-	if err := s.ArchiveTracker.Track(ctx, dbID, dbToken, req.GetKey(), jid, p.Cid()); err != nil {
+	if err := s.ArchiveTracker.Track(ctx, dbID, dbToken, req.GetKey(), jid, p.Cid(), owner); err != nil {
```

Comment:
Need to pass the owner PK in as part of the saved archive info so that we call pull the owner and ffs instance info out later when we are monitoring the archive